### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2690

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+1.5.0 --- 2021-07-19
+--------------------
+
+* Added support for django 3.0, 3.1 and 3.2
```
Comment (nedbat) [inline CHANGELOG.rst:20]: We don't need Django 3.0 or 3.1, right? Can't we simplify by just keeping Django 3.2?

**Review by nedbat — CHANGES_REQUESTED** [review]
We aren't going to use this with Django 3.0 or 3.1, so we can simplify, unless I am missing something?

**Comment by Jawayria** [comment]
> We aren't going to use this with Django 3.0 or 3.1, so we can simplify, unless I am missing something?
we need to add support for Django 3.0 and 3.1 and remove any deprecation warnings raised by them as these warnings may cause failures with Django3.2. We should make sure that the fixes we make work with Django 3.0, 3.1 and 3.2

File: .travis.yml [hunk 2]
```diff
@@ -16,6 +16,12 @@ matrix:
       env: TOXENV=django31-drf311 AFTER_SUCCESS=codecov
     - python: '3.8'
       env: TOXENV=django32-drf311 AFTER_SUCCESS=codecov
+    - python: '3.8'
+      env:
+        - COMMAND="make quality"
+        - PIPEXTRA="-r requirements/quality.txt"
+        - DEPLOY_PYPI=true
```
Comment (mraarif) [inline .travis.yml]: we don't need this variable since there's a separate Github workflow handling PyPi deployments

**Review by awais786 — APPROVED** [review]

**Review by nedbat — APPROVED** [review]

**Pull request merged**